xml/System.Text/StringRuneEnumerator.xml

```diff
@@ -219,8 +220,8 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <summary>Releases all resources used by the current <see cref="T:System.Text.StringRuneEnumerator" /> instance.</summary>
+        <remarks>This method performs no operation and produces no side effects.</remarks>
```

Comment:
Should this remark go at the end of the summary instead?

For example, look at [this PR](https://github.com/dotnet/dotnet-api-docs/pull/2803/files#diff-69e2ff9cb3ca19ddcb83db53484b00ceR785). We had to implement methods for the GroupCollection class because of the interfaces it inherits from, but some of them are not supported because the class is read-only. So in their summary, we mention what the function is supposed to do, but then we mention that it will always throw NotSupportedException.

Reply:
This method is supported. It no-ops. The text in this PR is a combination of the _summary_ from [`CharEnumerator.Dispose`](https://docs.microsoft.com/en-us/dotnet/api/system.charenumerator.dispose) (for the part about releasing resources) and the _remarks_ from [`GC.KeepAlive`](https://docs.microsoft.com/en-us/dotnet/api/system.gc.keepalive) (for the part about performs no operation and produces no side effect).

Reply:
I think it's fine as-is

Reply:
Ok no problem. The main reason why I was suggesting moving it to summary is so people could see that text in IntelliSense (remarks are not visible).